doc/authentication-methods/client-certificate.md

```diff
@@ -11,6 +11,17 @@ A server must request the certificate from a client, so you'll need to enable `v
 
 Please check [Listener modules](../advanced-configuration/Listener-modules.md#client-to-server-c2s-ejabberd_c2s) page for more information or simply follow the examples at the end of this section.
 
+### Properly configure `ejabberd_cowboy` listener
+
+SASL EXTERNAL authentication is also possible for WebSocketSecure and BOSH connections over HTTPS.
+Similarly to `ejabberd_c2s` case, the server must request the certificate from the client.
+In this case it's enabled by adding following options to `ssl` option of `ejabberd_cowboy` :
```

Comment:
In this case it's enabled by adding the following options to `ssl` option of `ejabberd_cowboy`: